extensions/amp-viewer-integration/0.1/messaging.js

```diff
@@ -288,7 +288,9 @@ export class Messaging {
       delete this.waitingForResponse_[requestId];
       if (message.error) {
         this.logError_(TAG + ': handleResponse_ error: ', message.error);
-        pending.reject(new Error(message.error));
+        const errorMsg = 'Request "${' + message.name + '} failed: ${' +
```

Comment:
Sorry, I mean either `${}` to be used in back-ticks, which are automatically expanded to values - string interpolation. If you use manual string concat, you don't need `${}`.